src/app/_tests/mocks.ts

```diff
@@ -60,10 +60,13 @@ const standardTypeValues = () => {
 const createText = (text: string): Text => {
```

Comment:
not something that you have to do here, just letting you know that we do expose `createTest*()` methods for several of the model objects, and the ones that are missing should be easy to expose/add.

They do have defined default values for all the parameters in Kotlin, but that may not be exported correctly to TypeScript.

```typescript
class Text extends org.cru.godtools.shared.tool.parser.model.Content {
    static createTestText(parent?: any/* org.cru.godtools.shared.tool.parser.model.Base */, text?: string, ...
}
```

Reply:
It seems like this function doesn't work properly on TS, but maybe I'm doing something wrong.

```
org.cru.godtools.shared.tool.parser.model.Text.createTestText({
    text: text
})
 ```

Returns the below object. it registers the values I input, but then doesn't set the text to those values.

```
{
  "u39_1": {
    "text": "Have you heard of the four spiritual laws?",
  },
  ...
  "text": "",
  ...
}
```